build/render.js

```diff
@@ -1,23 +1,24 @@
 import { readFile } from "node:fs/promises";
 
 import path from "node:path";
-import { fileURLToPath } from "node:url";
 
-import { render as distRender } from "../dist/ssr/index.js";
+import { BUILD_OUT_ROOT } from "./env.js";
 
-const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const { render: distRender } = /** @type {import("../entry.ssr.js")} */ (
+  await import(path.resolve(BUILD_OUT_ROOT, "static", "ssr", "index.js"))
```

Comment:
Silly question: If `static/ssr/index.js` is in BUILD_OUT_ROOT, does this mean we publish this file?

Reply:
Yes: it's true at the moment too, we don't need to, but there shouldn't be any harm in doing so. I'll file an issue to output this somewhere else.